server/server.go

```diff
@@ -620,3 +614,13 @@ func (s *Server) leaderRedirect(w http.ResponseWriter, r *http.Request) {
 	}
 	http.Redirect(w, r, u+r.URL.Path, http.StatusTemporaryRedirect)
 }
+
+func (s *Server) handleSnapshotThreshold(count, index uint64) {
+	if count > s.snapConf.snapshotAfter {
```

Comment:
I'd prefer if this check was moved outside of the function -- only call it if count is greater than the threshold. It's easier to follow the code that way, as well as explicitly uint-test the function.